File: src/AudioraMobile/app/src/main/java/com/audiora/app/models/RegisterRequest.java
package com.audiora.app.models;

public class RegisterRequest {
    private String name;
    private String email;
    private String password;
    private String confirmPassword;
    private String birthDate;
    private String country;

    public RegisterRequest(String name, String email, String password,
                           String confirmPassword, String birthDate, String country) {
        this.name            = name;
        this.email           = email;
        this.password        = password;
        this.confirmPassword = confirmPassword;
        this.birthDate       = birthDate;
        this.country         = country;
    }
}

File: src/Audiora.Application/DTOs/Request/RegisterRequest.cs
using Audiora.Domain.Enums;

namespace Audiora.Application.DTOs.Request;

public class RegisterRequest
{
    public string Name { get; set; } = string.Empty;
    public string Email { get; set; } = string.Empty;
    public string Password { get; set; } = string.Empty;
    public string ConfirmPassword { get; set; } = string.Empty;
    public DateTime BirthDate { get; set; }
    public string? Country { get; set; }
    public UserRole Role { get; set; } = UserRole.FreeUser;
}